airbyte-integrations/connector-templates/destination-python/requirements.txt

```diff
@@ -0,0 +1 @@
+-e .
```

Comment:
Since we are fully on setup.py, we can get rid of the `requirements.txt`?

Reply:
removed